gitbook-docs/market/overview.md

```diff
@@ -1,63 +1,84 @@
-The Proof Market version represented here is beta. We expect constant changes to the below as per the feedback we receive from participants.
+The Proof Market is currently in its beta version.
+We expect there will be continuous changes in it following the feedback we receive from the customers.
 {% endhint %}
 
-## Spot vs Futures
+## Spot vs. futures
 
-Every marketplace has a set of instruments. There are buyers and sellers in the market for each of these instruments. Example: London Stock Exchange allows users to buy/sell stocks of companies, or Binance allows users to buy/sell Bitcoin- these are known as spot markets as you own the commodity.&#x20;
+Every marketplace has a set of instruments.
+The market has buyers and sellers for each of these instruments.
+For example, London Stock Exchange allows users to buy and sell stocks of companies, Binance allows users
+to buy and sell Bitcoins — these are known as spot markets because you own the commodity.
 
-Exchanges also allow buying/selling of a future price of the instrument. Example: Binance allows users to buy/sell future values of Bitcoin via perpetuals - these are known as derivative markets as you do not own the underlying instrument but are just speculating on its price.
+Exchanges also allow buying and selling a future price of the instrument, like Binance enables users
+to buy and sell future values of Bitcoin via perpetual contracts.
+These are called derivative markets, as you do not own the underlying instrument
+but are just speculating on its price.
 
-\=nil;'s proof market is more on the lines of a Spot Market rather than a speculative value of proof generation.
+`\=nil;` Proof Market is more on the lines of a Spot Market rather than a speculative
+value of proof generation.
 
 {% hint style="info" %}
-The beta of the proof market is centralised.
+The beta version of the Proof Market is centralized.
 {% endhint %}
 
+## Instruments/circuits
 
-
-## Instruments/Circuits
-
-As mentioned above, markets tend to arrive at a fair value for an instrument through interactions between buyers and sellers. In =nil's proof markets, the instruments are Circuits (initially generated by the zkLLVM). These circuits are serialised byte codes created by the proof system, which was compiled with the zkLLVM compiler.&#x20;
+Markets tend to arrive at a fair value for an instrument through interactions between buyers and sellers.
+In `=nil;` Proof Market, the instruments are circuits — serialized byte codes created by the proof system,
+compiled with the zkLLVM compiler.
+To publish a circuit to the Proof Market, you should prepare a statement and attach
+all additional metadata to it.
+See the [prepare statement](cmd-reference/statement.md/#prepare-statement) section.
 
 {% hint style="info" %}
-The instruments will require approval by =nil;Foundation to trade on Proof Market.
+To be traded on the Proof Market, the instruments will require approval by `=nil;` Foundation.
```

Comment:
We've already defined instruments as statements/circuits, let's use it here?

Reply:
Done